.github/linters/sun_checks.xml

```diff
@@ -150,7 +149,6 @@
     <!-- See https://checkstyle.org/config_coding.html -->
     <module name="EmptyStatement"/>
     <module name="EqualsHashCode"/>
-    <module name="HiddenField"/>
```

Comment:
why ? Please comment this and write the reason why did u comment this. don't remove any line from this fine

Reply:
Please, check it again!

Reply:
again where is the fix of this ?

Reply:
added it and commented it.
we removed it because of initializing a variable with same name of param